apps/src/templates/EligibilityConfirmDialog.jsx

```diff
@@ -0,0 +1,98 @@
+/** @file Confirm Dialog for Maker Discount Codes */
+import React, {Component} from 'react';
+import i18n from "@cdo/locale";
+import BaseDialog from "./BaseDialog";
+import DialogFooter from "./teacherDashboard/DialogFooter";
+import Button from "./Button";
+
+const styles = {
+  subtitle: {
+    fontSize: 16,
+  }
+};
+
+export default class EligibilityConfirmDialog extends Component {
+  state = {
+    signature: "",
+    activeButton: false,
+  };
+
+  verifyResponse = () => {
+    if (document.getElementById('verify1').checked &&
+        document.getElementById('verify2').checked &&
+        document.getElementById('verify3').checked &&
+        this.state.signature !== ""){
+      this.setState({activeButton : true});
+    } else { // if someone unchecks their responses
+      this.setState({activeButton: false});
+    }
+  }
+
+  setSignature = (value) => {
+    this.setState({signature : value});
+    this.verifyResponse();
+  }
+
+  render() {
+    return (
+      <BaseDialog
+        useUpdatedStyles
+        uncloseable
+        isOpen = {true}
+        assetUrl={() => ''}
+        style={{padding:20}}
+      >
+        <h2>{i18n.getCode()}</h2>
+        <div style={styles.subtitle} >{i18n.verifyStatementsforCode()}</div>
+        <br/>
+        <form>
+          <label style={styles.boxStyle}>
+            <input
+              type="checkbox"
+              id="verify1"
+              onChange={this.verifyResponse}
+            />
+            {i18n.verifyStudentCount()}
+          </label>
+          <label>
+            <input
+              type="checkbox"
+              id="verify2"
+              onChange={this.verifyResponse}
+            />
+            {i18n.verifyYear()}
+          </label>
+          <label>
+            <input
+              type="checkbox"
+              id="verify3"
+              onChange={this.verifyResponse}
+            />
+            {i18n.verifySingleCode()}
+          </label>
+          <br/>
+          <label>
+            <div><b>{i18n.verifySignature()}</b>  {i18n.typeName()}</div>
+            <input
+              value=""
+              style={{width: 98 + '%', height: 40}}
```

Comment:
nit: I would put this in our styles const. Also, you can just do `width: '98%'` at this point. No need to concat a number and string.

Also, not sure if your intention is two characters of whitespace before typeName, but if so, I'd double check it's working correctly as React is sometimes tricky about stripping whitespace.